english/gameofthrones/characters/shae.md

```diff
@@ -0,0 +1 @@
+This is westeros and people here wanted to hurt me and they will look for any weakness they could find. I love you and I don;t want us hurt. It would be better if you would leave westeros for ive. You will have your own house, servants and what not. You could start a new life all together, says tyrion to shae.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.